java/protoc/pom.xml

```diff
@@ -8,7 +8,7 @@
   </parent>
   <groupId>com.google.protobuf</groupId>
   <artifactId>protoc</artifactId>
-  <version>3.21.2-rc1</version>
+  <version>3.21.2</version>
```

Comment:
seems weird for this to be the specified version on the main branch

Reply:
Yeah I left it at that from testing and recently realized I had never changed it back, which is why it's changing now. All of the other files have 3.21.2.